azure-kusto-data/tests/test_token_providers.py

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License
+import os
+import unittest
+import webbrowser
+from abc import ABC
+
+from azure.kusto.data._token_providers import *
+
+
+KUSTO_URI = "https://thisclusterdoesnotexist.kusto.windows.net"
+PUBLIC_AUTH_URI = "https://login.microsoftonline.com/"
+TOKEN_VALUE = "little miss sunshine"
+
+TEST_AZ_AUTH = False  # enable this in environments with az cli installed, and make sure to call 'az login' first
+TEST_MSI_AUTH = False  # enable this in environments with MSI enabled and make sure to set the relevant environment variables
+TEST_DEVICE_AUTH = False  # User interaction required, enable this when running test manually
+
+
+class MockProvider(TokenProviderBase, ABC):

```

Comment:
Both TokenProviderBase and ABC?

Reply:
I'm new to Python Inheritance, and was just as surprised when I got this 
![image](https://user-images.githubusercontent.com/59868016/100362453-1bd6da80-3004-11eb-84bc-0364b47ab9f0.png)

Reply:
That's weird, all other child classes doesn't have it as a superclass, the only difference I see is you used there:
`import abc`
and here:
`from abc import ABC`